lib/instructors.js

```diff
@@ -1,16 +1,16 @@
-import fs from "fs"
-import { compose, curry, map, pick, find } from "lodash/fp"
+import fs from 'fs'
+import {compose, curry, map, pick, find, uniqBy} from 'lodash/fp'
 
 const mapProps = compose(map, pick)
 const findPick = curry((selector, props, data) =>
-  pick(props, find(selector, data))
+  pick(props, find(selector, data)),
 )
 
-const instructorsPath = "./data/instructors.json"
+const instructorsPath = './data/instructors.json'
 
 function readInstructors() {
   const rawdata = fs.readFileSync(instructorsPath)
-  return JSON.parse(rawdata)
+  return uniqBy('slug', JSON.parse(rawdata))
```

Comment:
something is weird in the data and there were dupes that caused issues. Might be related to the issue you described @johnlindquist ?

Reply:
I investigated and found the dupes, but I think it's just from how we're doing the `loadStaticDataToFiles.js` pagination or something.

They do not appear as dupes in blazer queries or in the graphql request